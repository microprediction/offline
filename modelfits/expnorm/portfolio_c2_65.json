{"g1": 0.0301136446603009, "g2": 5.960940504479591, "logK": 0.9036284962539478, "loc": 0.04948920859465696, "logScale": -0.2465087342649308}