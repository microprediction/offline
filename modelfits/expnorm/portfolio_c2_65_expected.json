{"g1": 0.3516250733315409, "g2": 13.469937184955828, "logK": 0.8443776164637417, "loc": -0.12226987424874765, "logScale": -1.92970617199103}